docs/src/lib/tensors.md

```diff
@@ -24,23 +24,38 @@ TrivialTensorMap
 TrivialTensor
 ```
 
-## Specific `TensorMap` constructors
+## `TensorMap` constructors
 
+### General constructors
+
+A general `TensorMap` can be constructed by specifying its data, codmain and domain in one
+of the following ways:
```

Comment:
Perhaps highlight that the datastructure that the user gives depends on the symmetries of the codomain and domain ?

In particular 
Complexspaces -> single dense array
Symmetric spaces -> fusion tree

Reply:
I don't explain it in the .md file directly, but I think this should be clear from the docstrings I added (which is indeed not obvious since there's was docs preview (yet)). In particular, the docstring for the constructor from a dense array states how this works with and without symmetries,
https://github.com/Jutho/TensorKit.jl/blob/d671313509d90b634fa1609a1f497c9597af7644/src/tensors/tensor.jl#L313-L344
and the assignment using fusion trees (if this is what you meant) is in the `setindex!` docstring
https://github.com/Jutho/TensorKit.jl/blob/d671313509d90b634fa1609a1f497c9597af7644/src/tensors/tensor.jl#L616-L627